.github/workflows/release-nym-vpn-desktop.yml

```diff
@@ -174,6 +192,39 @@ jobs:
           ls -la target/${{ env.ARCH_TARGET }}/release/bundle || true
           ls -la target/${{ env.ARCH_TARGET }}/release/bundle/* || true
 
+      - name: Move update bundle artifacts (Linux)
+        if: contains(matrix.os, 'ubuntu')
+        shell: bash
+        # ja on ubuntu `fd` is specifically named `fdfind` …
+        run: |
+          echo "moving update bundle and signature into ${{ env.UPDATE_BUNDLE_DIR }}"
+          rm -rf ${{ env.UPDATE_BUNDLE_DIR }} || true
+          mkdir ${{ env.UPDATE_BUNDLE_DIR }}
+          fdfind -I 'nym-vpn.*.AppImage.tar.gz$' -x mv '{}' '${{ env.UPDATE_BUNDLE_DIR }}/update_x86_64.AppImage.tar.gz'
+          fdfind -I 'nym-vpn.*.AppImage.tar.gz.sig$' -x mv '{}' '${{ env.UPDATE_BUNDLE_DIR }}/update_x86_64.AppImage.tar.gz.sig'
+          ls -la ${{ env.UPDATE_BUNDLE_DIR }}
+
+      - name: Move update bundle artifacts (Mac)
```

Comment:
Let's try to deduplicate these two steps, there was a lot of effort put into this wf to try to unify steps where possible, and these two look very similar